src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/TaskEntityToCamundaReconfigureInputDefMapper.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.services;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.ReportingPolicy;
+import org.mapstruct.factory.Mappers;
+import uk.gov.hmcts.reform.wataskmanagementapi.domain.camunda.CamundaReconfigureInputVariableDefinition;
+import uk.gov.hmcts.reform.wataskmanagementapi.entity.TaskResource;
+
+@Mapper(componentModel = "spring", unmappedTargetPolicy = ReportingPolicy.IGNORE)
+
+public interface TaskEntityToCamundaReconfigureInputDefMapper {
+    TaskEntityToCamundaReconfigureInputDefMapper INSTANCE =
+        Mappers.getMapper(TaskEntityToCamundaReconfigureInputDefMapper.class);
```

Comment:
I guess the above comment about naming applies here too.

Reply:
updated